### DISCUSSION TITLE
WeaveVM announcement feedback 🗣️

### DISCUSSION THREAD

**Discussion by charmful0x** [weaveVM/rfcs#3] [General]
Welcome to the WeaveVM (WVM) discussion forum on Github! Your feedback, questions, and suggestions are highly valued as we strive to enhance the capabilities of WeaveVM.

## About WeaveVM

WeaveVM (WVM) is the first EVM-compatible protocol engineered to manage complex, large-scale data computation without the high gas fees typically associated with on-chain state storage on EVM chains. WVM is also the first EVM-compatible protocol that fully relies on Arweave for DA, enabling atomicity of source code and data storage on the same ledger. WVM incorporates the principles of [VACP](https://docs.wvm.dev/discover-weavevm/weavevm-and-vacp), bolstering its ability to handle complex data structures efficiently and gaslessly.

## Useful Links
- Official Website: https://wvm.dev
- Technical Documentation: https://docs.wvm.dev
- Follow us on Twitter/X: https://x.com/weavevm

Your engagement and input are instrumental in shaping the evolution of WeaveVM. Thank you for being a part of this journey!

**Comment by boyuanx** [comment]
Very excited, how can we get in touch?

**Reply by AceLuodan** [reply]
Very excited,can i help?

**Comment by Cryptocu105** [comment]
very excieted. Is there discord guys?

**Comment by B0DH1i** [comment]
very exicieted are there any community to join ıf u want ı can modarate this

**Comment by jamra13** [comment]
I’m happy to see something  is cooking. I saw darwin.ar’s tweet with metamask photo and get excited :) Hope I can participate and contribute!

**Comment by UDUMA9** [comment]
I'll be back with some feedback

**Comment by glogo2** [comment]
I have a few questions:
How does WeaveVM's "digestible format" for storing historical state compare to traditional archive node storage in terms of efficiency and scalability?
Are there risks of data loss or censorship under WeaveVM's storage model? How is data integrity and accessibility guaranteed across different Arweave storage providers?
What are the expected performance implications (latency, throughput) for accessing historical data on WeaveVM compared to traditional archive nodes?
